pkg/Library/Inspector/Atoms/ObjectInspector.js

```diff
@@ -54,7 +54,12 @@ renderProp(prop, parent, customInspectors, state) {
     }
   };
 },
-chooseTemplate({store: {type, values, range}, value}, isEditing, customInspectors) {
+primaryType(type) {
```

Comment:
Ideally all type munging is done via functions from the `TypeMatcher` module

Reply:
i'm curious - does order matter in the type (like is it from more specific to less specific or something)?

in ObjectInspector, we use `type` to:
- choose a template
- see if the type is not a primitive, then default to `connection`
both are display properties, so shouldn't directly be represented in TypeMatcher, i think, so i'm not fully sure how to express this from TypeMatcher perspective...

but what i did here is just (randomly) use the first type, i realize it's a hack, i should at least add a todo...

Reply:
In my head the types were ordered, but I don't think I ever formalized it (or really decided).

Best -> Least?